docs/reference-cmta.md

```diff
@@ -9,20 +9,19 @@ sidebar_label: cmta
 
 The available commands and their function is as follows:
 
-- [`studentify`](#cmta-studentify): generate a studentified artifact
-- [`linearize  `](#cmta-linearize): generate a linearized artifact
-- [`delinearize`](#cmta-delinearize): reflect the changes made in a linearized artifact back into the corresponding main repository
-- [`renum      `](#cmta-renum): renumber exercises in a main repository
-- [`dib        `](#cmta-dib): duplicate a selected exercise in a main repository and insert it before that exercise
+- [`studentify             `](#cmta-studentify): generate a studentified artifact
+- [`linearize              `](#cmta-linearize): generate a linearized artifact
+- [`delinearize            `](#cmta-delinearize): reflect the changes made in a linearized artifact back into the corresponding main repository
+- [`renumber-exercises     `](#cmta-renumber-exercises): renumber exercises in a main repository
+- [`duplicate-insert-before`](#cmta-duplicate-insert-before): duplicate a selected exercise in a main repository and insert it before that exercise
 
 The remainder of this section describes these commands in further detail.
 
-
 ### cmta studentify
 
 #### Synopsys
 
-`   cmta studentify [-fg] <Main repo> <studentified repo parent folder>`
+`   cmta studentify [-fgh] -m <Main repo> -s <studentified repo parent folder> [-c <config-file>]`
```

Comment:
should we align the target folder argument? i.e. just make it `-d` for every command... then it's easier to remember, even though it means something different for the different commands?

Reply:
@thinkmorestupidless 
Shall I move ahead an submit a PR for that?

Reply:
Went ahead and changed everything to utilise `-d`.
See #206